Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -655,7 +664,7 @@ protected Sheet createSheet() {
          * If the type of the artifact represented by this node dictates the
          * addition of the source content's unique path, add it to the sheet.
          */
-        if (Arrays.asList(SHOW_UNIQUE_PATH).contains(artifactTypeId)) {
+        if (artifactType != null && artifactType.getCategory() == Category.ANALYSIS_RESULT) {
```

Comment:
I just had another though.  Would all artifacts of Category type ANALYSIS_RESULT be AnalysisResult objects?  Can't you just do an instanceof instead?

Reply:
Unfortunately, I don't think so.  I think `getBlackboardArtifacts` returns a `BlackboardArtifact`.  That would be nice if we could.